universal/ubi8/Dockerfile

```diff
@@ -187,6 +187,7 @@ RUN dnf -y module enable container-tools:rhel8 && \
     dnf -y update && \
     dnf -y reinstall shadow-utils && \
     dnf -y install podman buildah skopeo fuse-overlayfs
+RUN echo 'alias docker=podman' >> /home/user/.bashrc
```

Comment:
It would, but I would probably rebase my PR off of this one and change it to

`RUN echo 'alias docker=podman' >> ${PROFILE_EXT}` as my PR makes the UDI image [not change the default ~/.bashrc](https://github.com/devfile/developer-images/pull/115/commits/da3316b717349505314ac6b3c8ce215536afc211#diff-9f21bef4669d474b6ddef409447be53ae91c94cb50b6df9345149a0e89fb5391R22-R24).